<docs>
	<doc for="EntryPointNotFoundException.EntryPointNotFoundException3">
		<summary>
			<para>Initializes a new instance of the <see cref="System.EntryPointNotFoundException"/> class with serialized data.</para>
		</summary>
		<param name="info">The object that holds the serialized object data.</param>
		<param name="context">The contextual information about the source or destination.</param>
		<remarks>
			<para>This constructor is called during deserialization to
      reconstitute the exception object transmitted over a stream. For more
      information, see <see topic="cpconSerialization"/>.</para>
		</remarks>
		<seealso topic="cpconSerialization"/>
	</doc><doc for="EntryPointNotFoundException">
		<summary>
			<para>The exception that is thrown when an attempt to load a class fails
      due to the absence of an entry method.</para>
		</summary>
		<remarks>
			<para> Represents the error that occurs when an attempt to load 
      an <see cref="System.Reflection.Assembly"/> fails due to the absence of an entry
      point.</para>
			<para> For a list of initial property values for an instance of <see cref="System.EntryPointNotFoundException"/>,
   see the <see cref="System.EntryPointNotFoundException()"/> constructors.</para>
		</remarks>
		<seealso cref="System.Exception"/>
		<seealso topic="cpconHandlingThrowingExceptions"/>
	</doc><doc for="EntryPointNotFoundException.EntryPointNotFoundException">
		<overload>
			<para>Initializes a new instance of the <see cref="System.EntryPointNotFoundException"/>
class.</para>
		</overload>
		<summary>
			<para>Initializes a new instance of the <see cref="System.EntryPointNotFoundException"/> class.</para>
		</summary>
		<remarks>
			<para>This constructor initializes the <see cref="System.Exception.Message"/> 
property of the new instance to a system-supplied message that describes the
error, such as "Entry point was not found." This message takes into account the
current system culture.</para>
			<para>The following table shows the initial property values for an instance of
<see cref="System.EntryPointNotFoundException"/>.</para>
			<list type="table">
				<listheader>
					<term>Property</term>
					<description>Value</description>
				</listheader>
				<item>
					<term><see cref="System.Exception.InnerException"/></term>
					<description>A 
      null reference (<see langword="Nothing"/> in Visual Basic).</description>
				</item>
				<item>
					<term><see cref="System.TypeLoadException.Message"/></term>
					<description>The localized error message string.</description>
				</item>
			</list>
		</remarks>
	</doc><doc for="EntryPointNotFoundException.EntryPointNotFoundException1">
		<summary>
			<para>Initializes a new instance of the <see cref="System.EntryPointNotFoundException"/>
class with a specified error message.</para>
		</summary>
		<param name="message">The error message that explains the reason for the exception.</param>
		<remarks>
			<para>The content of <paramref name="message"/> is intended to be 
   understood by humans. The caller of this constructor is required to ensure that
   this string has been localized for the current system culture.</para>
			<para>The following table shows the initial property values for an instance of
<see cref="System.EntryPointNotFoundException"/>.</para>
			<list type="table"><listheader>
					<term>Property</term>
					<description>Value</description>
				</listheader>
				<item>
					<term><see cref="System.Exception.InnerException"/></term>
					<description>A 
   null reference (<see langword="Nothing"/> in Visual Basic).</description>
				</item>
				<item>
					<term><see cref="System.TypeLoadException.Message"/></term>
					<description>The error message string.</description>
				</item>
			</list>
		</remarks>
	</doc><doc for="EntryPointNotFoundException.EntryPointNotFoundException2">
		<summary>
			<para>Initializes a new instance of the <see cref="System.EntryPointNotFoundException"/>
class with a specified error message and a reference to the
inner exception that is the cause of this exception.</para>
		</summary>
		<param name="message">The error message that explains the reason for the exception.</param>
		<param name="inner">The exception that is the cause of the current exception. If the <paramref name="inner"/> parameter is not a null reference (<see langword="Nothing"/> in Visual Basic), the current exception is raised in a <see langword="catch"/> block that handles the inner exception.</param>
		<remarks>
			<para>An exception that is thrown as a direct result of a
      previous exception should include a reference to the previous exception in
      the <see cref="System.Exception.InnerException"/> property. The
   <see cref="System.Exception.InnerException"/> property returns 
      the same value that is passed into the constructor, or a null reference
      (<see langword="Nothing"/> in Visual Basic) if the <see cref="System.Exception.InnerException"/>
      property does not supply the inner exception value to
      the constructor. </para>
			<para>The following table shows the initial property values for an instance of
   <see cref="System.EntryPointNotFoundException"/>.</para>
			<list type="table"><listheader>
					<term>Property</term>
					<description>Value</description>
				</listheader>
				<item>
					<term><see cref="System.Exception.InnerException"/></term>
					<description>The inner exception reference.</description>
				</item>
				<item>
					<term><see cref="System.TypeLoadException.Message"/></term>
					<description>The error message string.</description>
				</item>
			</list>
		</remarks>
		<seealso cref="System.Exception"/>
		<seealso topic="cpconHandlingThrowingExceptions"/>
	</doc></docs>
